src/daemons/rhsmcertd.c

```diff
@@ -35,6 +35,13 @@
 #include <libintl.h>
 #include <locale.h>
 
+typedef enum {
+    LOG_LEVEL_ERROR = 8,
+    LOG_LEVEL_WARNING = 4,
+    LOG_LEVEL_INFO = 2,
+    LOG_LEVEL_DEBUG = 1
+} LOG_LEVEL;
+
```

Comment:
Debug code like that works just fine with incremental integer values, and the small snippet I wrote works just fine with the changes to the debug macros you did. There is no need to have bit shifting here, since the values are not used as flags, but rather only one element is set at once.

Reply:
My code works too. There is no need to change it.

Reply:
Using a bitfield when multiple values for flags are not used makes this confusing for experienced C programmers. Since it is used as simple value without multiple options at the same time, please drop the power-of-2 values.

Reply:
I see both of your points. I'd personally vote for the incremental integer numbers unless there is a need for the powers of 2. @jirihnidek Do you see us needing to bit shift log values in the future? If not, can we please switch to the incremental integer values? This is great work and I look forward to it being pulled in!